project/Dependencies.scala

```diff
@@ -40,7 +40,7 @@ object Dependencies {
   object Compile {
     // Compile
 
-    val camelCore     = "org.apache.camel"            % "camel-core"                   % "2.13.4" exclude("org.slf4j", "slf4j-api") // ApacheV2
+    val camelCore     = "org.apache.camel"            % "camel-core"                   % "2.17.1" exclude("org.slf4j", "slf4j-api") // ApacheV2
```

Comment:
We have bin-compat rules documented here:
http://doc.akka.io/docs/akka/snapshot/common/binary-compatibility-rules.html
Though with Camel it is a bit tricky... Would be nice to upgrade it before
3.0 :-/

-- 
Konrad `ktoso` Malawski
Akka <http://akka.io> @ Lightbend <http://lightbend.com>

On 30 June 2016 at 11:48:28, Ian Clegg (notifications@github.com) wrote:

In project/Dependencies.scala
<https://github.com/akka/akka/pull/20851#discussion_r69103810>:

> @@ -40,7 +40,7 @@ object Dependencies {
>    object Compile {
>      // Compile
>
> -    val camelCore     = "org.apache.camel"            % "camel-core"                   % "2.13.4" exclude("org.slf4j", "slf4j-api") // ApacheV2
> +    val camelCore     = "org.apache.camel"            % "camel-core"                   % "2.17.1" exclude("org.slf4j", "slf4j-api") // ApacheV2

If you are referring to binary compatibility between camel 2.13.4 and
2.17.1 with respect to the types required only by akka - it is time
consuming, but possible to check.
Otherwise, if you refer to compatibility across all types and classes in
camel core - I think it is difficult to assert unless camel dev know
otherwise? This is my first PR, aside from common sense, what the hard
rules around binary compatibility for Akka

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/akka/akka/pull/20851/files/715f66ad8331cdcd5ba058287318a27293425e9b#r69103810>,
or mute the thread
<https://github.com/notifications/unsubscribe/AAHYkwTSjH3Euzqtr7u2Ul_NbQUessqpks5qQ5DpgaJpZM4JAaMP>
.